terraform/spec/fixtures/projects/private_module_with_dir_suffix/main.tf

```diff
@@ -0,0 +1,4 @@
+module "my-module" {
+  source  = "registry.example.com/org/name/provider//sub/dir/module"
```

Comment:
I don't know `terraform` very well... is the double slash `//` in this path expected/desired?

Reply:
Yeah, this is to select a subfolder within the module source. https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories